pages/contact.js

```diff
@@ -1,7 +1,101 @@
-import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Use React-icons instead of @fortawesome, I've already added react-icons to package.json so it's ready to use. Also inside react-icons you get access to font-awesome icons